components/ui/ProjectPublishingChecklist.vue

```diff
@@ -1,4 +1,18 @@
 <template>
+  <div v-if="$auth.user && showInvitation" class="universal-card information invited">
+    <h2>Invitation to join project</h2>
+    <p>
+      You've been invited be a member of this project with the role of '{{ currentMember.role }}'.
+    </p>
+    <div class="input-group">
+      <button class="iconified-button brand-button" @click="acceptInvite()">
+        <CheckIcon />Accept
+      </button>
+      <button class="iconified-button danger-button" @click="declineInvite()">
+        <CrossIcon />Decline
+      </button>
+    </div>
+  </div>
```

Comment:
why is this in the publishing checklist? why not just in the main mod page?

Reply:
I could rename the component, it's more of a project page header I suppose? It should show in both settings and main project pages, if it was just in the page it would be duplicated code. Or I could make it its own component? but don't really see any reason to separate it

Reply:
yeah renaming would probs be good